Source/WebCore/platform/graphics/ColorTypes.h

```diff
@@ -711,6 +721,11 @@ struct RGBA {
     {
     }
 
+    explicit RGBA(uint8_t red, uint8_t green, uint8_t blue, uint8_t alpha)
```

Comment:
Why is this needed?

Reply:
It's not if we serialize the uint32_t instead.